src/classes/STG_InstallScript.cls

```diff
@@ -65,6 +65,8 @@ global without sharing class STG_InstallScript implements InstallHandler {
             Version previousVersion = context.previousVersion();
 
             disablePreferredEmailEnforcement(previousVersion);
+            
+            populdateAdminRecordTypeSetting(previousVersion);
```

Comment:
super simple but "populate" has been mispelled here. We could move forward but its a 5 second change so I figure we might as well fix it.